src/core/ext/filters/client_channel/lb_policy/ring_hash/ring_hash.cc

```diff
@@ -37,6 +37,8 @@
 #include "absl/strings/string_view.h"
 #include "absl/types/optional.h"
 
+#include "src/core/lib/iomgr/resolved_address.h"
```

Comment:
This header reordering change looks incorrect.  I think clang-format is being confused by the `#define XXH_INLINE_ALL` on line 42.

Reply:
Thanks! Reverted.